app/controllers/events_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: false
+
+# class Events
+class EventsController < ApplicationController
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = Event.new(events_paramas)
+
+    if @event.save
+      redirect_to '/views/events/index'
```

Comment:
You can use the prefixes that rails gives you with resources.

See  the documentation at:
https://guides.rubyonrails.org/routing.html#resource-routing-the-rails-default